SimpleITK/utilities/fft.py

```diff
@@ -22,20 +22,22 @@
 def fft_based_translation_initialization(
     fixed: sitk.Image, moving: sitk.Image
 ) -> sitk.TranslationTransform:
-    """
-    Perform fast Fourier transform based normalized correlation to find the translation of the maximize correlation
-     between the images.
+    """Perform fast Fourier transform based normalized correlation to find the translation which maximizes correlation
+    between the images.
 
     If the moving image grid is not congruent with fixed image ( same origin, spacing and direction ), then it will be
     resampled onto the grid defined by the fixed image.
 
     Efficiency can be improved by reducing the resolution of the image or using a projection filter to reduce the
     dimensionality of the inputs.
 
-    :param fixed: A SimpleITK image object
+    :param fixed: A SimpleITK image object.
+    :type fixed: sitk.Image
```

Comment:
Duplication should be avoided. The arguments are already typed in the signature. Perhaps this module would help:
sphinx-autodoc-typehints

Reply:
types removed from docstring